wallet/client.go

```
@@ -58,6 +76,10 @@ func (c *grpcClient) connect() error {
 		_, err = blockchainClient.GetBlockchainInfo(c.ctx,
 			&pactus.GetBlockchainInfoRequest{})
 		if err != nil {
+			if err := conn.Close(); err != nil {
```

I suggest to ignore error here:
```
_ = conn.Close()
```